tests/test.py

```diff
@@ -434,6 +434,21 @@ def test_sensi_power_transfer(self):
         self.assertEqual(0.25664095577626006, s['BBE2AA1  FFR3AA1  1']['DE -> FR'])
         self.assertEqual(0.10342626899874961, s['BBE2AA1  FFR3AA1  1']['NL'])
 
+    def test_variant(self):
+        n = pp.network.create_eurostag_tutorial_example1_network()
+        self.assertEqual('InitialState', n.get_working_variant_id())
+        n.clone_variant('InitialState', 'WorkingState')
+        n.set_working_variant('WorkingState')
+        self.assertEqual('WorkingState', n.get_working_variant_id())
```

Comment:
would be nice to add something to show that we can change the variant independently of each other:
like change some values (for example the position of a switch) on WorkingState, check it has been taken into account, then change back the working variant to 'InitialState' and check that the position of the switch has not changed here.